js/package.json

```diff
@@ -27,18 +27,18 @@
     "extension": "src/lab_extension"
   },
   "devDependencies": {
-    "css-loader": "^3.5.3",
-    "eslint": "^6.8.0",
-    "eslint-config-prettier": "^6.10.1",
-    "fs-extra": "^4.0.2",
-    "json-loader": "^0.5.4",
-    "prettier": "^2.0.2",
-    "rimraf": "^2.6.1",
-    "style-loader": "^0.23.1",
-    "webpack": "^4.41.6",
-    "webpack-cli": "^3.3.11"
+    "css-loader": "^3.6.0",
+    "eslint": "^7.4.0",
+    "eslint-config-prettier": "^6.11.0",
+    "fs-extra": "^9.0.1",
+    "json-loader": "^0.5.7",
+    "prettier": "^2.0.5",
+    "rimraf": "^3.0.2",
+    "style-loader": "^1.2.1",
+    "webpack": "^4.43.0",
+    "webpack-cli": "^3.3.12"
   },
   "dependencies": {
-    "@jupyter-widgets/base": "^2 || ^3"
```

Comment:
I think that the `^2 || ^3` is what allows simultaneous support of jupyterlab 1 and 2.

Reply:
oh wow. i sure didn't see the other comment already saying as much before i wrote that...